api/server.py

```diff
@@ -391,33 +391,6 @@ def download_crash_id(crash_id):
     return jsonify(message=url)
 
 
-def isValidUser(user_dict):
```

Comment:
per our discussion [in slack](https://austininnovation.slack.com/archives/CM9MK950S/p1736890649856289) and this [issue comment](https://github.com/cityofaustin/atd-data-tech/issues/20391#issuecomment-2558161548). we're no longer validating these fields which existed on the Auth0 `idToken`.